.copier-answers.yml

```diff
@@ -0,0 +1,24 @@
+# Do NOT update manually; changes here will be overwritten by Copier
+_commit: v1.14.0
```

Comment:
Hi! I have updated the repo configuration and fixed a couple of parameters in https://github.com/qrtl/server-env/pull/1, could you please have a look and maybe merge it? Thanks!

Reply:
@SirTakobi not a blocker, you can open another PR against OCA when this is merged.